### PR TITLE
Add files for BC verification

### PR DESCRIPTION
**Pull request by uditagar**
### PR Checklist

* [x] test-datasets is a publicly accessible github repo. The contents of this PR can be made public.
* [x] The naming and organization of the dataset I am adding follows https://github.com/KatanaGraph/test-datasets#organization
* [x] If I am adding an RDG, its directory contains a (import/migrate/generate) script, and a README describing the RDGs usage. https://github.com/KatanaGraph/test-datasets#the-following-scripts-are-conditionally-present-for-each-rdg-one-of-the-following-options-must-be-present
* [x] If I am uprevving the storage_format_version of the rdgs in this repo, I have followed https://github.com/KatanaGraph/test-datasets#how-to-uprev-the-rdgs-in-this-repo

### List of folks that should be notified of this PR. Please don't remove these. 
@scober
@aneeshdurg

### CODE REVIEW

File: misc_datasets/verify/betweennesscentrality.ldbc003.sourceFile [hunk 1]
```diff
@@ -0,0 +1,100 @@
+4113
```
Comment (roshandathathri) [inline misc_datasets/verify/betweennesscentrality.ldbc003.sourceFile:1]: These sources are a parameter to the app. 

For queries, we store the parameters in the main repo:
https://github.com/KatanaGraph/katana-enterprise/tree/master/data
I would suggest you add a new folder for analytics there but I'm not opposed to storing it in test-datasets.

**Review by roshandathathri — COMMENTED** [review]
LGTM too but please wait for the other reviewers

**Review by aneeshdurg — APPROVED** [review]
LGTM - I don't think we have any clear guidelines around misc_datasets, so I'm on the fence about keeping the sourceFile here vs in enterprise. I like the fact that the source and expectation are close together here, but it also seems different from other things in this repo

**Comment by gliga** [comment]
I vote for @roshandathathri 's suggestion.  Test config, lists, etc can go to enterprise (data) and input/output graphs could be in test-datasets.

**Pull request closed without merging**